app/services/geetest_verifier.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'digest'
+require 'net/http'
+
+class GeetestVerifier
+  def initialize
+    @api = 'http://api.geetest.com'
+    @validate_path = '/validate.php'
+    @register_path = '/register.php'
+    @geetest_id = ENV['GEETEST_ID']
+    @geetest_key = ENV['GEETEST_KEY']
```

Comment:
This secrets should go to barong secret storage. Look over the app to see an example